doc/user-guides/acrn-shell.rst

```diff
@@ -104,10 +107,10 @@ function ``acpi_idle_do_entry``.
 
    system map information
 
-dumpmem
-=======
+dump_host_mem
+=============
 
-``dumpmem mem_address`` provides the specified memory target data such as
+``dump_host_mem hva,length`` provides the specified memory target data such as
```

Comment:
Do you need to comma in between `hva` and `length`? If not, please remove it

Reply:
have removed it, thanks